test/test-fs-copyfile.c

```diff
@@ -125,6 +125,11 @@ TEST_IMPL(fs_copyfile) {
   r = uv_fs_copyfile(NULL, &req, src, src, 0, NULL);
   ASSERT(r == 0);
   uv_fs_req_cleanup(&req);
+  /* Verify that the src file did not get truncated. */
+  r = uv_fs_stat(NULL, &req, src, NULL);
+  ASSERT(r == 0);
+  ASSERT(req.statbuf.st_size == 12);
```

Comment:
Use the newer macros?

Reply:
Why `req.statbuf.st_size` should be equal to 12? I don't get it :(

Reply:
@iandrc The source file was created to be 12 bytes in size and this assert checks that the size of the source file was not altered.
https://github.com/libuv/libuv/blob/452cd0a3222e41ad5f7e8419fdcd21a3fb902b8f/test/test-fs-copyfile.c#L124

Reply:
@richardlau updated.